### PR TITLE
New package: bvi-1.4.0

### PR DESCRIPTION
**Pull request by oliver-cfc**

### CODE REVIEW

File: srcpkgs/bvi/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'bvi'
+pkgname="bvi"
+version="1.4.0"
+revision="1"
+build_style="gnu-configure"
+makedepends="ncurses-devel"
+short_desc="A display-oriented editor for binary files"
+maintainer="oliver <ohemming@gmail.com>"
+license="GPL-1"
```
Comment (Vaelatern) [inline srcpkgs/bvi/template]: Seriously?

Comment (oliver-cfc) [inline srcpkgs/bvi/template]: Just thought I'd add pkgs not in the repo that I use fairly regularly.  If
it's not appropriate (and if not please let me know why so I won't make the
mistake again) then feel free to just close the request.

Comment (dominikh) [inline srcpkgs/bvi/template]: According to the COPYING file bundled with bvi, it's licensed under version 2 of the GPL, not version 1.

Comment (Vaelatern) [inline srcpkgs/bvi/template]: Yes. I don't know of any projects that still use GPLv1.

**Comment by Vaelatern** [comment]
Since I was here anyway, eb2babb is the merged code, with the license fixed.

**Comment by oliver-cfc** [comment]
Thanks all

**Pull request closed without merging**